File: app/src/main/java/net/tlalka/android/fiszki/model/dto/WelcomeDto.java
package net.tlalka.android.fiszki.model.dto;

import org.parceler.Parcel;

@Parcel
public class WelcomeDto {

    protected String message;

    public WelcomeDto() {
    }

    public WelcomeDto(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }
}

File: app/src/main/kotlin/net/tlalka/fiszki/model/dto/WelcomeDto.kt
package net.tlalka.fiszki.model.dto

import org.parceler.Parcel
import org.parceler.ParcelConstructor

@Parcel(Parcel.Serialization.BEAN)
class WelcomeDto {

    var message: String = ""

    @ParcelConstructor
    constructor()

    constructor(message: String) {
        this.message = message
    }
}
